src/options_page.js

```diff
@@ -69,6 +69,7 @@ const OPTIONAL_PERMISSIONS_URLS = {
       '/*',
   ),
   'github': ['https://github.com/*'],
+  'stackoverflow': ['https://stackoverflow.com/*', 'https://www.stackoverflow.com/*'],
```

Comment:
Please change to `stack-overflow`, and for the JS variables below use `stackOverflow`